ibmcloud/image/push.sh

```diff
@@ -82,7 +82,7 @@ while image_id=$(ibmcloud is images --output=json --visibility=private | jq -r "
     fi
 done
 
-echo -e "\nUploading $image_path to cloud object stroage at "$cos_bucket" with key $object_key\n"
```

Comment:
maybe stroage is a typo?

Reply:
I suspect here the original author wanted $cos_bucket enclosed by double-quotes.